addons/web/static/src/js/services/crash_manager.js

```diff
@@ -297,6 +306,27 @@ var CrashManager = core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin,
             data: {debug: ""}
         });
     },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {string} message
+     * @param {string} title
+     * @param {string} subtitle
+     * @param {Object} options
+     */
+    _displayWarning: function (message, title, subtitle, options) {
+        return new WarningDialog(this, {
+            ...options,
```

Comment:
crash in Edge, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

The creation of an object with ... is not supported (es2018)

Reply:
Thx for the review ! : https://github.com/odoo/odoo/pull/35820